lua/super-commit/git/commands.lua

```diff
@@ -6,10 +6,15 @@ function M.show_command_output(bufnum, command)
   vim.api.nvim_buf_set_lines(bufnum, 0, -1, false, output_table)
 end
 
-local commands = {}
-commands[3] = "git status"
-commands[4] = "git diff --name-only --cached"
+function M.show_diff_cmd_output(bufnum, path)
+  M.show_command_output(bufnum, "git diff --cached ".. path)
+end
 
-M.commands = commands
+local explanation = "To show git diff of files, select the file path in the window below, and press Enter."
```

Comment:
Make variable names more descriptive. 
For example:
```suggestion
local diff_keymap_msg = "To show git diff of files, select the file path in the window below, and press Enter."
```

Reply:
This is solved.